six/include/six.h

```diff
@@ -48,7 +48,7 @@ class Six
 
     // Public Const API
     virtual bool isInitialized() const = 0;
-    virtual const char *getPyVersion() const = 0;
+    virtual py_info_t *getPyInfo() = 0;
```

Comment:
Why did we drop the constness?

Reply:
Because we now use `_fetchPythonError` which is not const.